HHS/Head-Start-TTADP
# [TTAHUB-810] Fix For Touchpoints Survey

## Description of change

Fix For Touchpoints Survey

We will now open the TP survey in a new tab to prevent an origin mismatch from CORS.

This allows us to keep the security policy in place while keeping the survey.

## How to test

Go to the AR landing page and click the "Please leave feedback" button. The survey should open in a new window and allow you to submit successfully.

## Issue(s)

* https://ocio-jira.acf.hhs.gov/browse/TTAHUB-810


## Checklists

### Every PR


~- [ ] API Documentation updated~
~- [ ] Boundary diagram updated~
~- [ ] Logical Data Model updated`
~- [ ] [Architectural Decision Records](https://adr.github.io/) written for major infrastructure decisions~

### Production Deploy


### After merge/deploy
